src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.tsx

```diff
@@ -31,11 +31,14 @@ export const Healthcheck = (props: HealthcheckProps) => {
     const {data, loading, wasLoaded, error} = useSelector((state: any) => state.healthcheckInfo);
     const {autorefresh} = useSelector((state: any) => state.schema);
 
-    const fetchHealthcheck = useCallback(() => {
-        if (fetchData) {
-            dispatch(getHealthcheckInfo(tenant));
-        }
-    }, [dispatch, fetchData, tenant]);
+    const fetchHealthcheck = useCallback(
+        (isForcedFetch = false) => {
+            if (fetchData || isForcedFetch) {
+                dispatch(getHealthcheckInfo(tenant));
+            }
+        },
+        [dispatch, fetchData, tenant],
+    );
 
     useAutofetcher(fetchHealthcheck, [fetchHealthcheck], autorefresh);
```

Comment:
i'd just moved the test of `fetchData` here instead of introducing `isForcedFetch`

Reply:
I found your solution much cleaner, so i applied it